src/Andoromeda.Kyubey.Portal/wwwroot/js/candlestick/index.js

```diff
@@ -1,2 +1,96 @@
-
-
+﻿
+
+var kyubeyExchangeCandlestick = (function () {
+    var defaultActiveCycle = 240;
+    //var tokenId = window.tokenId;
+
+    var addCandlestickClass = function (fatherDom, dom) {
+        [...fatherDom.getElementsByTagName('span')].forEach(function (item) {
+            item.className = ''
+        })
+        dom.className = 'active'
+    }
+    var addInternalChangedEventListener = function () {
+        var _this = this;
+        var intervalDom = document.getElementById('interval')
+        // 切换产品周期
+        intervalDom.addEventListener('click', function (e) {
+            // e.target.dataset.value 这个就是获取的产品的周期
+            widget.chart().setResolution(e.target.dataset.value)
+
+
+            // 这个函数不用看，我为了样式好看 写一个添加删除class
+            addCandlestickClass(intervalDom, e.target)
+        }, false)
+    }
+    var initCandlestick = function () {
+        var _this = this;
+        window.TradingView.onready(function () {
+            // chartConfig 在chartConfig.js里面
+            // 给chartConfig添加展示周期
+            chartConfig.interval =defaultActiveCycle;
+            // 给chartConfig添加展示产品
+            chartConfig.symbol = `${window.tokenId}/EOS`// index_market
+
+            // 初始化 TradingView
+            widget = new window.TradingView.widget(chartConfig)
+
+            widget && widget.onChartReady && widget.onChartReady(function () {
+                //// 这是k线图 展示的 7日均线和30日均线。

```

Comment:
Can you please remove the commented code?

Reply:
fixed